src/tmx/TmxUtils/src/telemetry/TelemetryDeserializer.cpp

```diff
@@ -0,0 +1,16 @@
+#include "TelemetryDeserializer.h"
+
+namespace tmx::utils::telemetry
+{
+    boost::property_tree::ptree TelemetryDeserializer::stringToJson( const string& jsonString){
```

Comment:
This function would benefit from better naming. This function reads a JSON string into a boost property tree. A name like readJson or jsonToPtree makes more sense to me since I property tree is not JSON itself.

Reply:
even deserializeJson would make more sense to me. If you have a better alternative I would also be open to that.